monkey/tests/unit_tests/monkey_island/cc/services/test_authentication_service.py

```diff
@@ -35,34 +36,47 @@ def get_user_credentials(self, username: str) -> UserCredentials:
 
 
 @pytest.fixture
-def mock_repository_encryptor():
+def mock_repository_encryptor(autouse=True):
```

Comment:
You don't need to pass these to the functions if you're using autouse.

Reply:
When you return an object you need to pass them. I don't know how this even got here.